src/main/java/io/github/jsonSnapshot/Snapshot.java

```diff
@@ -69,11 +81,38 @@ private String getRawSnapshot(Collection<String> rawSnapshots) {
         return null;
     }
 
+    private Object ignorePaths(Object input) {
+        DocumentContext context = JsonPath.using(new GsonJsonProvider()).parse(gsonForIgnoringPaths.toJsonTree(input));
```

Comment:
Considered alternative: Using a GSON [`ExclusionStrategy`](https://google.github.io/gson/apidocs/com/google/gson/ExclusionStrategy.html). However, such a strategy has no notion of the path to a field. The methods in the interface only allow finding the type of the containing class and the name of each field.

Reply:
Consider changing JSON parse lib to Jackson. We changed default JSON parser lib from GSON to Jackson due to issue #9